medusa/providers/torrent/json/t411.py

```diff
@@ -205,6 +216,8 @@ def search(self, search_strings, age=0, ep_obj=None):
         if not self.login():
             return results
 
+        self._get_terms()
```

Comment:
The terms should be a module-global that is initialized as None (or `{}`).  The `_get_terms()` method should then only populate `if not terms`.  If someone should need to update terms in the future as part of some other logic it becomes as simple as `terms = None` and it will update on the next call to `_get_terms`